spec/spec_helper.rb

```diff
@@ -59,6 +59,10 @@ def photo_fixture_name
   @photo_fixture_name = File.join(File.dirname(__FILE__), "fixtures", "button.png")
 end
 
+def jwks_file_path
+  @jwks_file = "../../spec/fixtures/jwks.json"
```

Comment:
Is this creating a file for every client application? This looks like it should go into the database.